prog/types.go

```diff
@@ -622,6 +633,23 @@ func (t *StructType) isDefaultArg(arg Arg) bool {
 	return true
 }
 
+func (t *StructType) Alignment() uint64 {
```

Comment:
If we have TypeCommon.TypeAlign, I think we don't need this. TypeCommon.Alignment should return the correct value for StructType as well.